odoo/cli/i18n.py

```diff
@@ -0,0 +1,228 @@
+        subparsers = self.parser.add_subparsers(
+            dest='subcommand', required=True,
+            help='Subcommands help')
+
+        self.import_parser = subparsers.add_parser(
+            'import',
+            help="Import i18n files",
+            description="Imports provided translation files",
+            formatter_class=SubcommandHelpFormatter,
+        )
+        self.export_parser = subparsers.add_parser(
+            'export',
+            help="Export i18n files",
+            description="Exports language files into the i18n folder of each module",
+            formatter_class=SubcommandHelpFormatter,
+        )
+        self.loadlang_parser = subparsers.add_parser(
+            'loadlang',
+            help="Load languages",
+            description="Loads languages",
+            formatter_class=SubcommandHelpFormatter,
+        )
+
+        for parser in (self.import_parser, self.export_parser, self.loadlang_parser):
+            parser.add_argument(
+                '-c', '--config', dest='config',
+                help="use a specific configuration file")
+            parser.add_argument(
+                '-d', '--database', dest='db_name', default=None,
+                help="database name, connection details will be taken from the config file")
+
+        self.import_parser.add_argument(
+            'files', nargs='+', metavar='FILE',
+            help=f"files to be imported. Allowed extensions: {', '.join(IMPORT_EXTENSIONS)}\n")
+        self.import_parser.add_argument(
+            '-w', '--overwrite', action='store_true',
+            help="overwrite existing terms")
+        self.import_parser.add_argument(
+            '-l', '--language', dest='language', metavar='LANG', required=True,
+            help="language code")
+
+        self.export_parser.add_argument(
+            '-l', '--languages', dest='languages', nargs='+', default=['pot'], metavar='LANG',
+            help="list of language codes, 'pot' for template (default)")
+        self.export_parser.add_argument(
+            'modules', nargs='+', metavar='MODULE',
+            help="modules to be exported")
+        self.export_parser.add_argument(
+            '-o', '--output', metavar="FILE", dest='output',
+            help=(
+                "output only one file with translations from all provided modules\n"
+                f"allowed extensions: {', '.join(EXPORT_EXTENSIONS)},"
+                " '-' writes a '.po' file to stdout\n"
+                "only one language is allowed when this option is active"
+            ),
+        )
+
+        self.loadlang_parser.add_argument(
+            '-l', '--languages', dest='languages', nargs='+', metavar='LANG',
+            help="List of language codes to install")
+
+    def run(self, cmdargs):
+        try:
+            parsed_args = self.parser.parse_args(args=cmdargs)
+        except ValueError as e:
+            self.parser.print_help()
+            sys.exit(f'\n{e}\n')
```

Comment:
In what cases this exception occurs? IIRC argparse already print the usage and emits a `SystemExit` upon failure